price_matching_system/templates/user.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<div class="row mt-5">
+    <div class="col-md-4 mx-auto p-5" style="min-width: 500px; border: solid black">
+        <h1>Account Info</h1>
+        <hr />
+        <div class="mx-auto mt-5 p-2">
+            <label class="form-label" for="email">Email</label>
+            <input id="email" class="form-control form-control-lg" value="{{ g.user['Email'] }}" readonly>
```

Comment:
can we not use inputs in the display page?
it confuses the user whether they can edit or not

Reply:
changed to <div>